backend/lib/fetch/census.py

```diff
@@ -0,0 +1,57 @@
+"""Fetch census data by service area."""
+import collections
+import logging
+
+from backend.lib.fetch import representative_points
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: Include this information in the `fetch_all_service_areas` response.
```

Comment:
What do you mean?

Reply:
This doesn't really make sense to me long-term as its own API endpoint. 

Ideally we would fetch service areas at the same time we fetch information about them -- rural vs. urban, census data, etc. The frontend can ignore parts of the response during codegen, then use the same endpoint as needed when the application is running.

Reply:
Ok maybe add some more details into what you have in mind